splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -83,6 +83,7 @@ else if(v1==v2)
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1953), new UpgradeScriptForRemoveUnusedIndexInSYSFILESTable(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1954), new UpgradeScriptToInvalidateStoredStatement(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1959), new UpgradeScriptForTriggerMultipleStatements(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,1,0, 1961), new UpgradeScriptForAddDefaultToColumnViewInSYSVW(sdd,tc));
```

Comment:
If we target this week's build, you will need to set this to 1962

Reply:
The version has been updated